DeveloperConsole.cs

```diff
@@ -21,12 +21,13 @@ public class DeveloperConsole : MelonMod
         private Canvas DevConsole;
         static Library LC_Lib = new Library();
         PlayerControllerB Player;
+        Terminal Terminal;
```

Comment:
Terminal does not exist in context

Reply:
I'm not exactly following. Upon building the code, it finds the Terminal GameObject and compiles it.